arches/app/views/graph.py

```diff
@@ -449,19 +468,13 @@ def post(self, request, graphid=None):
             return JSONResponse(ret, force_recalculation=True)
         except GraphValidationError as e:
             return JSONErrorResponse(e.title, e.message, {"status": "Failed"})
-        except PublishedModelError as e:
-            return JSONErrorResponse(e.title, e.message)
 
     @method_decorator(group_required("Graph Editor"), name="dispatch")
     def delete(self, request, graphid):
         if self.action == "delete_node":
             data = JSONDeserializer().deserialize(request.body)
             try:
                 graph = Graph.objects.get(graphid=graphid)
```

Comment:
Clicking the delete node button just after publishing a graph (before dismissing the success banner) raises a 500. This seems unlikely, but I could see it happening if a user made a mistake and didn't bother dismissing the banner.


https://github.com/archesproject/arches/assets/38668450/84226bee-dd78-4379-af4b-3f7601f4b1cc